src/app/detail/ui/Sidebar.tsx

```diff
@@ -88,7 +88,7 @@ export const Sidebar = ({
         borderLeft: (theme) => `1px solid ${theme.color.borders.border2}`,
         height: '100vh',
         display: showSidebar ? 'block' : 'none',
-        width: matches && showSidebar ? '100vw' : '25vw',
+        width: matches && showSidebar ? '100vw' : { sm: '16vw', md: '20vw' },
```

Comment:
I think the width of the sidebar doesn't match the Figma design. According to the design, the sidebar occupied 29.5% of the total screen width which is equivalent to 366px, whereas in this PR the width of sidebar is very small i.e. ~ 336px.